integration/tests/cook/util.py

```diff
@@ -490,6 +490,10 @@ def minimal_group(**kwargs):
     return dict(uuid=str(make_temporal_uuid()), **kwargs)
 
 
+def default_submit_pool():
```

Comment:
Should have a comment here. Including saying why its OK for response to be None. Also consider inlining this into the caller to make the Noneness of a potential response more visible.

Reply:
Good call, added a docstring.